OpenUtau/Views/SingersDialog.axaml.cs

```diff
@@ -191,6 +192,27 @@ public partial class SingersDialog : Window, ICmdSubscriber {
             }
         }
 
+        void OnOpenReadme(object sender, RoutedEventArgs e) {
+            var viewModel = (DataContext as SingersViewModel)!;
+            if (viewModel.Singer != null) {
+                var readme = $@"{viewModel.Singer.Location}\readme.txt";
```

Comment:
Backslash only works on Windows. use `var readme = Path.Join(viewModel.Singer.Location,"readme.txt")` so that it is cross-platform.

Reply:
Thanks. Fixed.